receiver/mongodbreceiver/actual_metrics.json

```diff
@@ -0,0 +1,2006 @@
+{
```

Comment:
Is this intentional?

Reply:
Resolved in [dd1469a](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16316/commits/dd1469ad255ab3d2a931d418b4d626d8714b9ea2)